partcad/src/partcad/actions/part_actions.py

```
@@ -0,0 +1,142 @@
+    try:
+        shutil.copy(source_path, target_path)
+    except Exception as e:
+        raise ValueError(f"Failed to copy file from {source_path} to {target_path}: {e}")
+
+    # Add the part to the project
+    project.add_part(kind, str(target_path), config)
+    project.update_part_config(name, {"path": str(target_path)})
+    pc_logging.info(f"Part {name} imported and added successfully.")
+
+
+def convert_part_action(project, part_name, target_format):
```

Ruff (0.8.2)

Missing return type annotation for public function `convert_part_action`

Add return type annotation: `None`

(ANN201)